catalog/app/containers/Admin/Form.js

```diff
@@ -16,8 +17,8 @@ export function Field({ input, meta, errors, helperText, ...rest }) {
 
 const useCheckboxStyles = M.makeStyles({
   root: {
-    paddingBottom: 0,
-    paddingTop: 0,
+    marginBottom: -9,
+    marginTop: -9,
```

Comment:
I think we use units everywhere:

```
marginBottom: '-9px'
```

Reply:
well, our JSS setup uses pixels as default unit, so i think it's ok. the only place it may cause bugs is line-height.